addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -1007,7 +1017,7 @@ var FormRenderer = BasicRenderer.extend({
         this.$el.html($newContent);
         this.$el.toggleClass('o_form_nosheet', !this.has_sheet);
         if (this.has_sheet) {
-            this.$el.children().not('.oe_chatter')
+            this.$el.children().not('.o_FormRenderer_chatterContainer')
```

Comment:
Still web referencing mail stuff here, can't we fix it now that we have done the rest?